jenkins_pipelines/environments/common/pipeline-pull-request.groovy

```diff
@@ -206,7 +206,12 @@ def run(params) {
                         env.resultdir = "${WORKSPACE}/results"
                         env.resultdirbuild = "${resultdir}/${BUILD_NUMBER}"
                         env.tf_file = "susemanager-ci/terracumber_config/tf_files/Uyuni-PR-tests-env${env_number}.tf" //TODO: Make it possible to use environments for SUMA
-                        env.common_params = "--outputdir ${resultdir} --tf ${tf_file} --gitfolder ${resultdir}/sumaform"
+
+                        if (!params.terraform_parallelism) {
```

Comment:
What I already mentioned above in Oscar's suggestion. I think there is no possibility that this variable is empty or not set since we always set a value in terracumber. See comment above.

Reply:
I would just keep it to be on the safe side. That variable is coming from another file, so a developer could change it later on.